farmdata2_modules/fd2_tabs/resources/FarmOSAPI.spec.js

```diff
@@ -98,7 +100,7 @@ describe('API Request Function', () => {
         })
     })
     
-    context('test maping functions', () => {
+    context.only('test maping functions', () => {
```

Comment:
The .only should be removed here so that all tests will run in the version in the upststream.

Reply:
I have removed the .only. Thank you for pointing it out.